LICENSES/vendor/golang.org/x/mod/LICENSE

```diff
@@ -1,4 +1,4 @@
-= vendor/golang.org/x/crypto licensed under: =
+= vendor/golang.org/x/mod licensed under: =
 
```

Comment:
This is strange… does this mean we're missing the crypto license now even though we're still vendoring code from it?

Reply:
nm, this got dropped from `hack/tools/LICENSES/vendor/golang.org/x/crypto/LICENSE` but is still alive and well in `LICENSES/vendor/golang.org/x/crypto/LICENSE`